lib/api/controllers/adminController.js

```diff
@@ -113,7 +113,7 @@ class AdminController extends BaseController {
 
     return this.kuzzle.repositories.user.truncate(options)
       .then(deletedUsers => {
-        result.deleteUsers = deletedUsers;
+        result.deletedUsers = deletedUsers;
```

Comment:
I find it weird that such a change does not trigger a failing test somewhere.

Reply:
Yes me too :sweat: